packages/react-core/src/components/Toolbar/examples/Toolbar.md

```diff
@@ -239,6 +239,71 @@ class ToolbarSpacers extends React.Component {
 }
 ```
 
+### Sticky
+
+```js
+import React from 'react';
+import { Toolbar, ToolbarItem, ToolbarContent } from '@patternfly/react-core';
+import { Button, ButtonVariant, InputGroup, TextInput, Checkbox } from '@patternfly/react-core';
+import SearchIcon from '@patternfly/react-icons/dist/esm/icons/search-icon';
+
+const ToolbarItems = () => {
+  const [isSticky, setIsSticky] = React.useState(true);
+  const [showEvenOnly, setShowEvenOnly] = React.useState(true);
+
+  return (
+    <React.Fragment>
+      <Checkbox
+        label="Is toolbar sticky"
+        isChecked={isSticky}
+        onChange={setIsSticky}
+        aria-label="checkbox for is sticky"
+        id="isStickyCheckbox"
+      />
+      <div style={{ overflowY: 'scroll', height: '200px' }}>
+        <Toolbar
+          id="toolbar-spacers"
+          inset={{
+            default: 'insetNone',
```

Comment:
i think the example looks best when the value of the `inset` prop is:
```
inset={{default: 'insetNone'}}
```
What do you think?

Reply:
I agree, changed!